document-store/src/main/java/org/hypertrace/core/documentstore/postgres/Params.java

```diff
@@ -0,0 +1,151 @@
+package org.hypertrace.core.documentstore.postgres;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Holds the params that need to be set in the PreparedStatement for constructing the final PQL
```

Comment:
Please fix the comment. Looks like this has been copy/pasted from query-service

Reply:
Yes, will fix it